ceph/utils.py

```diff
@@ -353,8 +353,8 @@ def create_ceph_nodes(
                     # can be used only for specific OSD_SCENARIO
                     node_params["osd-scenario"] = node_dict.get("osd-scenario")
 
-                if node_dict.get("image-name"):
-                    node_params["image-name"] = node_dict.get("image-name")
+                if node_dict.get("image-name", {}).get("openstack"):
```

Comment:
I think it should be `node_dict.get("image-name"`.

Reply:
done